build-system/pr-check/checks.js

```diff
@@ -50,6 +51,7 @@ async function prBuildWorkflow() {
 
   if (buildTargetsInclude(Targets.PRESUBMIT)) {
     timedExecOrDie('gulp presubmit');
+    timedExecOrDie('gulp video-interface-list');
```

Comment:
This has nothing to do with our presubmit checks. I think it's more related to runtime changes. Can you move this to line 108, after `gulp performance-urls`?

Reply:
Unfortunately `Targets.RUNTIME` includes only `src/`, not `extensions/`. Happy to restructure target types if needed.

Reply:
That's not true, even though it appears so. It's only seeded with files from `src/`. On top of that, _every_ file that doesn't fall through to a different specialized target automatically falls through into `Targets.RUNTIME`. (It would be weird if changes to `extensions/` did not go through all runtime tests.)

https://github.com/ampproject/amphtml/blob/a0f71dfb0381e236d6c6be287549748838731467/build-system/pr-check/build-targets.js#L318-L320

Reply:
I see, I misunderstood initially. Changed.